code/game/objects/items/tcg/tcg.dm

```diff
@@ -0,0 +1,273 @@
+///A global list of cards, or rather changes to be applied to cards in the format
+
+GLOBAL_LIST_EMPTY_TYPED(card_list, /datum/card)
+
+/obj/item/tcgcard
+	name = "Coder"
+	desc = "Wow, a mint condition coder card! Better tell the Github all about this!"
+	icon = 'icons/obj/tcg.dmi'
+	icon_state = "runtime"
+	w_class = WEIGHT_CLASS_TINY
+	var/id = -1 //Unique ID, for use in lookups and storage, used to index the global datum list where the rest of the card's info is stored
+	var/flipped = 0
+
+/obj/item/tcgcard/Initialize(mapload, datum/card/temp)
+	. = ..()
+	name = temp.name
+	desc = temp.desc
```

Comment:
If spawned directly through the game panel this will runtime.
Could you handle that?

Reply:
Yep